src/overlay/template.jsx

```diff
@@ -22,11 +22,11 @@ export type OverlayProps = {|
         continueMessage? : string
     |},
     autoResize? : boolean,
-    hideCloseButton? : boolean
+    hideCloseButton? : boolean,
+    nonce? : string
```

Comment:
Can we make `nonce` required?
```suggestion
    nonce : string
```